### PR TITLE
WTF fixed

### PR DESCRIPTION
**Pull request by digi604**

### CODE REVIEW

File: src/db/query.js [hunk 1]
```diff
@@ -63,7 +64,10 @@ async function query(query, db = null, removeNull = false, source = 'modal') {
 
           for (let j = 0; j < e.columns.length; j++) {
             const column = e.columns[j]
-
+            let fieldData = res[j]
+            if (PLATFORM === "WINDOWS" && typeof fieldData == "string") {
+              fieldData = fieldData.split("''").join("'") // WTF!!!!
```
Comment (kouts) [inline src/db/query.js:69]: It would make sense to do a `replace(/''/g, '"');` here for that extra performance

File: src/db/query.js [hunk 2]
```diff
@@ -63,7 +64,10 @@ async function query(query, db = null, removeNull = false, source = 'modal') {
 
           for (let j = 0; j < e.columns.length; j++) {
             const column = e.columns[j]
-
+            let fieldData = res[j]
+            if (PLATFORM === "WINDOWS" && typeof fieldData == "string") {
```
Comment (kouts) [inline src/db/query.js:68]: Strict equality check `fieldData === "string"` should be fine in all browsers

**Review by elibolonur — CHANGES_REQUESTED** [review]
Updaye changelog please

File: src/db/query.js [hunk 3]
```diff
@@ -1,4 +1,5 @@
-import { fireEvent } from '../event'
+import {fireEvent} from '../event'
+import {PLATFORM} from "@/platform";
```
Comment (elibolonur) [inline src/db/query.js]: Also this alias will collide with the consumer project alias, this should be relative path.

**Review by elibolonur — CHANGES_REQUESTED** [review]

File: src/db/query.js [hunk 4]
```diff
@@ -63,7 +64,10 @@ async function query(query, db = null, removeNull = false, source = 'modal') {
 
           for (let j = 0; j < e.columns.length; j++) {
             const column = e.columns[j]
-
+            let fieldData = res[j]
+            if (PLATFORM === "WINDOWS" && typeof fieldData === "string") {
```
Comment (wojtiku) [inline src/db/query.js:68]: I think this is the place where a comment would be justified explaining why this code is here. Something like `// Single quotes become duplicated on Windows which leads to parsing errors.` WDYT?

**Review by wojtiku — APPROVED** [review]

File: src/db/query.js [hunk 5]
```diff
@@ -1,4 +1,5 @@
-import { fireEvent } from '../event'
+import {fireEvent} from '../event'
+import {PLATFORM} from "../platform";
```
Comment (kouts) [inline src/db/query.js:2]: Can we change this to `import {PLATFORM} from '../platform'`, sorry didn't catch it the first time.

File: src/db/query.js (hunk 4 above)
Comment (kouts) [reply src/db/query.js:68]: Can we change this to single quotes `if (PLATFORM === 'WINDOWS' && typeof fieldData === 'string') {`

File: src/db/query.js [hunk 6]
```diff
@@ -63,7 +64,10 @@ async function query(query, db = null, removeNull = false, source = 'modal') {
 
           for (let j = 0; j < e.columns.length; j++) {
             const column = e.columns[j]
-
+            let fieldData = res[j]
+            if (PLATFORM === "WINDOWS" && typeof fieldData === "string") {
+              fieldData = fieldData.replace(/''/g, "'")// WTF!!!!
```
Comment (kouts) [inline src/db/query.js:69]: Can we change this to single quotes as well `fieldData = fieldData.replace(/''/g, '\'') // WTF!!!!`

**Review by kouts — COMMENTED** [review]
Can we also replace some double quotes just to be aligned with the rest

File: src/db/query.js (hunk 5 above)
Comment (wojtiku) [reply src/db/query.js:2]: Another one we need to add to our linter

**Review by elibolonur — APPROVED** [review]

**Pull request merged**